src/applications/facility-locator/containers/FacilitesMap.jsx

```diff
@@ -443,6 +475,15 @@ const FacilitiesMap = props => {
 
     searchWithUrl();
 
+    if (navigator.geolocation) {
```

Comment:
Can we please add a comment here to indicate that this is a temporary solution that will be re-done once we have time to do a proper design and hardening cycles?

Reply:
(maybe make a reference to https://github.com/department-of-veterans-affairs/vets-website/pull/14963 as an example of how to do this properly with React...?)

Reply:
will do